openapi/generated/.openapi-generator-ignore

```diff
@@ -0,0 +1,9 @@
+package.json
+tsconfig.json
+.npmignore
+.gitignore
+README.md
+.travis.yml
+.openapi-generator-ignore
+.openapi-generator
+git_push.sh
```

Comment:
You should probably remove the `--ignore-file-override=.openapi-generator-ignore` parameter in the [generation script](https://github.com/jomarko/sandbox-ui/blob/PR-MGDOBR-420/scripts/generate_openapi_client.sh#L30) if the ignore file is moved to the output folder. Right?

Reply:
@kelvah good point, will do